src/useBobo.ts

```diff
@@ -0,0 +1,18 @@
+const SPOTIFY_URL = 'https://open.spotify.com/track/1jGATZ2aFc6yL5ycanq0iO';
+const YOU_TUBE_URL = 'https://www.youtube.com/watch?v=nn5hCEMyE-E';
+
+interface BoboAttrs {
+  spotifyUrl: string;
+  youTubeUrl: string;
+}
+
+const BOBO_ATTRS: BoboAttrs = {
+  spotifyUrl: SPOTIFY_URL,
+  youTubeUrl: YOU_TUBE_URL,
+};
+
+export function useBobo(): BoboAttrs {
+  return BOBO_ATTRS;
```

Comment:
It all looks good, but this PR states "provides the up-to-date links", but these links are hard-coded, not up-to-date. Another way to look at it is from our `CONTRIBUTING.md`:

> .. a proposal for a new hook will need to utilize the React Hooks API internally to be taken into consideration.

Stated another way, a React hook internally needs to use at least another React hook. Otherwise it is just a JavaScript function, not a React hook. If this hook would use other hooks to somehow fetch the up-to-date URLs, instead of them being hard-coded in the code, then this would be a real hook and a really good one, exactly something this library always needed and was missing all this time.

Reply:
> exactly something this library always needed and was missing all this time.

So glad you share the vision.  I appreciate the suggestions!  I originally intended for the links to be updated via the normal release cycle, e.g. handling URL rot via a patch release or something.  I worry that hardcoding a reference to a hosted service (BaaS, Bobo as a service, if you will) is more brittle as part of a utility library.  Perhaps we should consider making this hook consume `useState`, which would then conditionally return an audio or video link?

```jsx
// rough draft
const [{ medium, url }, updateBoboMedium] = useBobo('audio');
return (
  <div>
    <code>{ url }</code>
    <button onClick={() => updateBoboMedium('video')}>
      Render { medium === 'audio' ? 'Video' : 'Audio' } URL
    <button>
  </div>
);
```

Reply:
Third option: I could create an npm module with the sole purpose of storing the latest URLs to official "There is a Party" streams in a JSON file.  That module could be declared as a dependency.  Pros: updates would be patch releases, so we wouldn't need to explicitly update `react-use` in the event that a URL needs to be mutated.  Cons: module tax, plus it seems like a lot of work for a feature that will only need to be updated once every few years, if ever.

Reply:
The proposed `useState` alternative seems forced. We need something better, for example, it could use the `useAudio` hook:

```ts
const useBobo = () => useAudio({src});
```

Reply:
@streamich that would be the ultimate realization of this idea.  My only concern with creating an implementation of `useAudio` would be copyright.  It's not typical for a commercial artist to host DRM-free versions of their recordings that can be distributed by any random software developer.  We could try to get permission from DJ Bobo himself to grant a license to distribute "There is a Party" under `react-use`'s license if and only if it's done through the `useBobo` API.  DJ Bobo's art has given us so much, but I will ask him for this favor for the good of the project.  Without permission, though, I don't think we can ship something like this.